ubuntu-dev/supervisord.conf

```diff
@@ -7,7 +7,11 @@ user = user
 command = sudo /usr/sbin/sshd -D
 [program:c9sdk]
 user = user
-command = /home/user/.c9sdk/server.js --auth : --collab --listen 0.0.0.0 --packed --port 8089 --useBrowserCache --workspacetype=janitor -w /home/user
+command = /home/user/.c9sdk/server.js --auth : --collab --listen 0.0.0.0 --packed --port 8089 --useBrowserCache --workspacetype=janitor -w %(ENV_WORKSPACE)s
```

Comment:
Exciting, your `ENV WORKSPACE` variable looks much nicer than our current sed/replace!

But how does `%(ENV_WORKSPACE)` work? (Can we be sure this won't get templated at `COPY` time in `ubuntu-dev.dockerfile`, thus ignore subsequent changes in later Dockerfile instructions like in `thunderbird.dockerfile`?)

Reply:
It's a supervisord config syntax, which essentially pulls in the environment variable at runtime.

Reply:
Ok thanks. Note that `ENV` variables from Dockerfiles don't transpire to runtime environments like Cloud9/VNC terminals. Let's keep an eye out for workspace regressions.